docs/installation/openshift/Deploying_openshift-acme_on_OpenShift.md

```diff
@@ -0,0 +1,108 @@
+Deploying openshift-acme on OpenShift
+=====================================
+
+## Overview
+
+This document describes the process to deploy [openshift-acme](https://github.com/tnozicka/openshift-acme)
+to issue certificates for applications on OpenShift automatically
+using [PKI ACME Responder](Deploying_PKI_ACME_Responder_on_OpenShift.md).
```

Comment:
Same here. Make sure it is notated that a requirement is to have deployed `PKI ACME Responder` already in their own infra.

Reply:
OK